gapis/server/grpc.go

```diff
@@ -87,24 +87,25 @@ func (s *grpcServer) stopIfIdle(ctx context.Context, server *grpc.Server, keepAl
 	// no activity from the client in a contiguous N chunks of time.
 	// This avoids killing the server if the machine is suspended (where the
 	// client cannot send hearbeats, and the system clock jumps forward).
-	const maxMissCount = 3
-	idleTimeout /= maxMissCount
-	misses := 0
+	const waitTime = time.Second
```

Comment:
There is also no harm, more importantly - it is not expected - so it is good to notify the user that things seem unresponsive.  And give heads up that it will be killed.